src/config/message.js

```diff
@@ -0,0 +1,82 @@
+// import Event from "./event.js";
+// import { ClickEvent, ChooseEvent } from "./event.js";
+import Config from "./config.js";
+
+export default class Message {
+  constructor(message) {
+    this.msg = message;
+  }
+
+  display(parentDom) {
+    const newMessage = document.createElement("div");
+    newMessage.setAttribute("class", "buddy-message");
+    newMessage.innerHTML = `
+      <img src="${Config.BuddyProfilePicSrc}" class="buddy-chat-profile-pic" alt="buddy profile pic">
+      <p class="buddy-message-content">${this.msg}</p>
+    `;
+    parentDom.append(newMessage);
+  }
+
+  scrollToBottom() {
+    const chatContainerDom = document.querySelector(".chat-content");
+    chatContainerDom.scrollTop = chatContainerDom.scrollHeight;
+  }
+}
+
+export class EventMessage extends Message {
+  constructor(event) {
+    super();
+    this.msg = event.message;
+    this.event = event;
+  }
+
+  display(parentDom) {
+    const newMessage = document.createElement("div");
+    newMessage.setAttribute("class", "buddy-message");
+    newMessage.innerHTML = `
+      <img src="${Config.BuddyProfilePicSrc}" class="buddy-chat-profile-pic" alt="buddy profile pic">
+      <p class="buddy-message-content">${this.msg}</p>
+    `;
+    parentDom.append(newMessage);
+
+    if (!this.event.userOptions) {
+      return
+    };
+
+    this.event.displayOptions(parentDom);
+  }
+}
+
+export class NotifMessage extends Message{
+  constructor(type, config) {
+    super();
```

Comment:
```suggestion
```
This is to trigger the parent's class constructor. If you do not need it, then don't. In this case, you don't